react/SlideToggle/SlideToggle.less

```diff
@@ -1,60 +1,59 @@
 @import (reference) "~seek-style-guide/theme";
 
+@toggleHeight: @row-height * 5;
+@toggleWidth: 48px;
+
 .root {
-  line-height: @row-height * 5;
-  height: @row-height * 8;
-  padding: (@row-height * 2) 0;
+  height: @touchableTextHeight;
+  position: relative;
+  display: inline-flex;
+  align-items: center;
   cursor: pointer;
-  display: inline-block;
 }
 
 .switch {
   display: inline-flex;
-  position: relative;
+  cursor: pointer;
 }
 
 .input {
+  opacity: 0;
   position: absolute;
-  left: 10px;
-  top: 5px;
+  left: 1px;
   margin: 0;
+  width: @toggleWidth;
+  height: @toggleHeight;
+  cursor: pointer;
 
-  &:focus ~ .slider .slideButton::before {
+  &:focus ~ .switch .slider .slideButton::before {
```

Comment:
The focus style has gone missing, you probably need to remove this `::before` selector to be consistent with the rest of this PR.